ui/panels-plugin/package.json

```diff
@@ -14,7 +14,8 @@
   "dependencies": {
     "@formatjs/intl-numberformat": "^7.4.1",
     "@mui/material": "^5.4.0",
-    "@perses-ui/core": "^1.0.0",
+    "@perses-dev/core": "^0.1.0",
+    "@perses-dev/plugin-system": "^0.1.0",
     "date-fns": "^2.28.0",
     "echarts": "^5.3.0",
     "lodash-es": "^4.17.21",
```

Comment:
don't you want to consider the `react` dependency as a `peerDependency` too ? Like it is the case in others packages ?

Reply:
Yup, good call out. Will fix this.